### PR TITLE
3DMLoader: Update to support Rhino 8 files.

### PR DESCRIPTION
**Pull request by fraguada**
**Description**

This PR adds support for the Rhino 8 file format. Most work thus far has been in supporting PBR materials now available in openNURBS.

I've included updated libs, documentation, and slightly updated the example.

*This contribution is funded by [McNeel](https://rhino3dm.com)*

### CODE REVIEW

File: editor/js/Loader.js [hunk 1]
```diff
@@ -93,7 +93,14 @@ function Loader( editor ) {
 					loader.setLibraryPath( '../examples/jsm/libs/rhino3dm/' );
 					loader.parse( contents, function ( object ) {
 
+						object.name = filename;
+
 						editor.execute( new AddObjectCommand( editor, object ) );
+						console.log( object )
```
Comment (Mugen87) [inline editor/js/Loader.js:99]: Do you mind removing this line. We usually do not log entire objects in the browser console.

File: examples/jsm/libs/rhino3dm/rhino3dm.d.ts
[the remark this answers is not in the document]
Comment (Mugen87) [reply examples/jsm/libs/rhino3dm/rhino3dm.d.ts:1]: TypeScript type declaration files should be no part of the repo.

File: editor/js/Loader.js (hunk 1 above)
Comment (fraguada) [reply editor/js/Loader.js]: yep, oversight, fixed in next commit.

File: examples/jsm/libs/rhino3dm/rhino3dm.d.ts
[the remark this answers is not in the document]
Comment (fraguada) [reply examples/jsm/libs/rhino3dm/rhino3dm.d.ts:1]: nope. removed in next commit.

**Comment by fraguada** [comment]
thanks @Mugen87!

**Pull request merged**